src/templates/generic-story/generic-story.tsx

```diff
@@ -44,18 +44,22 @@ const Wrapper = styled.div`
  */
 export const GenericStory = ({ story, config }: CommonTemplateTypes) => {
   const footerText = get(config, ["publisherConfig", "publisher-settings", "copyright"], null);
+  let showPoweredByQt = get(config, ["opts", "featureConfig", "showPoweredByQt"], true);
```

Comment:
Don't pass this as prop to footer. Check inside footer component

Reply:
then we will have to pass config to footer.

Reply:
context. Check how other components use context